### ISSUE TITLE
Request for an API function to change playlist name

### ISSUE THREAD

**Issue by Rp70** [frelo/fvc#2]
I don't see we have an API function to rename a playlist. I tried:
```csharp
playlist.Name = "New name";
```
It seems to work but it just update the GUI instead of updating the catelog file. The change doesn't persist if I create a new playlist manually or restart the software.

Is there any way to change playlist name without waiting for the next software release?

Thank you!

**Comment by frelo** [comment]
Yes you are correct, there is no api to rename it (and we should add it).

A workaround now would be to just create a new playlist with the name you want, copy all clips from the old playlist to the new and then delete the old playlist.

The apis would be 
CreateVideoPlaylist( name )
GetPlaylistClip(...)
SetClipToPlaylist(...)
DeletePlaylist( playlist_id );

**Comment by Rp70** [comment]
It works! Thank you!
